solr/solr-ref-guide/modules/deployment-guide/pages/solr-on-hdfs.adoc

```diff
@@ -210,41 +210,6 @@ NRTCachingDirectory max cache size.
 +
 Pass the location of HDFS client configuration files - needed for HDFS HA for example.
 
-=== Kerberos Authentication Settings
```

Comment:
as I mentioned, I think this can stay

Reply:
Restored these changes....   I really don't have a sense one way or the other...   If it's not correct, hopefully some of our Hadoop HDFS users will speak up!